streaming/src/test/scala/org/apache/spark/streaming/ReceivedBlockHandlerSuite.scala

```diff
@@ -251,9 +377,20 @@ class ReceivedBlockHandlerSuite
     (blockIds, storeResults)
   }
 
+  /** Store block using a handler */
+  private def storeBlock(
```

Comment:
existing storeBlocks take Sequence of blocks and return Sequence of blockId and BlockResult. In my test cases I need to count messages in single Block. Probably can rename it to storeSingleBlock which will be much intuitive . Also I can generate random block id there also using generateBlockId